crt_portal/static/sass/custom/landing.scss

```diff
@@ -218,6 +218,10 @@ $arrow-height: 24px;
       display: inline-block;
     }
 
+    a:visited {
+      color: color('white');
+    }
+
```

Comment:
This looks good.  I mightttt consider narrowing the selector a bit to prevent it from causing side effects.  

Soooo you could do something like 

```
.crt-landing--section {
    a:visited {
      color: color('white');
    }
}
```
And see if that both gets the results you want without accidentally changing something on another page.